bases/br_bd_diretorios_brasil/escolas/table_config.yaml

```
@@ -0,0 +1,137 @@
+source_bucket_name: basedosdados-dev
+project_id_staging: basedosdados-dev
+project_id_prod: basedosdados-dev
```

tem que mudar aqui nao?
```suggestion
project_id_prod: basedosdados
```

---

Não, é `basedosdados-dev` mesmo.